frontend/__tests__/BuyParts.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { afterEach, beforeEach, expect, it } from 'vitest'
```

Comment:
Can this folder just be named `tests`? `__tests__` seems unnecessarily annoying.

Reply:
Resolved in [039b270](https://github.com/kitspace/kitspace-v2/pull/446/commits/039b270cbdfd8d6c0b5f0b270beed0bd23e54fee).

Reply:
Nice, but sorry, should have said `test` to match the processor.